### PR TITLE
Bpbj nlo m2200 100k

### PR DESCRIPTION
**Pull request by DaLu0**
Dears,
My name is Dalia Ramírez from Universidad Iberoamericana (México). We, together with my supervisor Cristina Oropeza, are working with the B2G group.
We would like to produce the samples for our process using the model VLQ_v4_4FNS_UFO, using as starting point the following process:
https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/PairVLQ_b13b13_Incl_NLO_narrow/PairVLQ_b13b13_Incl_NLO_narrow_M1000

Could you please help us by telling if there is something to be changed or modified.

Kind regards,
Dalia

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/bpbj_LO_M2200_100k_narrow_Single/bpbj_LO_M2200_100k_customizecards.dat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+set param_card mass 6000007 2200
+
```
[the remark this answers is not in the document]
Comment (DaLu0) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/bpbj_LO_M2200_100k_narrow_Single/bpbj_LO_M2200_100k_customizecards.dat:2]: when I try to remove the last line in every card, it is still there when I access to them again, is this correct or should it be removed?

[the remark this answers is not in the document]
Comment (DaLu0) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/bpbj_LO_M2200_100k_narrow_Single/bpbj_LO_M2200_100k_customizecards.dat:2]: Hello!
I was wondering if there is some other comments about these cards and about my previous questions?

Best, 
Dalia

**Comment by DaLu0** [comment]
@menglu21 Hi thanks for your comments, I made some changes and posted few comments.
I was wondering what else is needed for it to be approved?
Best,
Dalia

**Comment by menglu21** [comment]
Hi @DaLu0 it seems your changes are not shown here, could you please check they are pushed from your repo

**Comment by DaLu0** [comment]
Dear @menglu21,
Is there anything left for me to do?
I see the green banner saying that this branch has no conflicts with the base branch.

Best,
Dalia

**Comment by menglu21** [comment]
> Dear @menglu21, Is there anything left for me to do? I see the green banner saying that this branch has no conflicts with the base branch.
> 
> Best, Dalia

**Comment by menglu21** [comment]
> Dear @menglu21, Is there anything left for me to do? I see the green banner saying that this branch has no conflicts with the base branch.
> 
> Best, Dalia

@DaLu0 sorry i missed your update, btw, my previous comments don't show up correctly, but your update look fine to me. i'll merge if no comment from other people.

**Comment by DaLu0** [comment]
Dear @menglu21 thanks a lot for your reply.
Great to know, that everything seems fine, please let me know once it is merged.
Cheers

**Comment by DaLu0** [comment]
Dear @menglu21,
I wish you a great year!
I was wondering if the process has been merged?
Kind regards,
Dalia

**Comment by DaLu0** [comment]
Dear @menglu21  & @Saptaparna , 
I hope You are doing fine.
I was wondering if this process has been merged, I haven't had any feedback and I would like to know if there is a problem, or if there is anything I should do?. Right now I am just waiting for this to continue with my analysis.
Thanks a lot for your attention.
Best,
Dalia

**Comment by ph2632** [comment]
Dear GEN, 
@samarendran23 and I have questions on the cards [here](https://github.com/cms-sw/genproductions/blob/mg265UL/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/bpbj_LO_M2200_100k_narrow_Single/bpbj_LO_M2200_100k_proc_card.dat):
- Why "define mybp = bp bp~" is defined but not used?
- Does "b b~" corresonds to the VLQs? (as b-quark can not be present in the "VLQ_v4_4FNS_UFO-3rdL_noCKM")? Is this correct?
Thank you in advance

**Comment by DaLu0** [comment]
Dear @ph2632 ,
- This is a remnant from a previous version of the card, where we generated the process until the production of the bp. Then we changed it to generate the "final state" and so it was not necessary anymore. 
- No, b b~ are the SM quarks. Why do you say it can't be present in this model?
Thanks,
Dalia

**Pull request merged**